### PR TITLE
Convert Pkg.Object

### PR DESCRIPTION
**Pull request by olafurpg**

### CODE REVIEW

File: tests/converter/src/test/scala/org/scalameta/tests/LotsOfProjects.scala [hunk 1]
```diff
@@ -37,10 +37,11 @@ object LotsOfProjects extends ConverterSuite {
       Try(getConvertedMetaTree(code)) match {
         // Uncomment to investigate a specific error further.
         //          case scala.util.Failure(e: ConvertException)
-        //              if e.culprit.getClass.getSimpleName == "EmptyTree$" =>
+        //              if e.culprit.getClass.getSimpleName == "ModuleDef" =>
         //            println(file.githubUrl)
         //            val culprit = e.culprit.toString.lines.take(1).mkString
         //            e.printStackTrace()
+        //            ???
```
Comment (xeno-by) [inline tests/converter/src/test/scala/org/scalameta/tests/LotsOfProjects.scala]: Why change the commented lines?

Comment (olafurpg) [inline tests/converter/src/test/scala/org/scalameta/tests/LotsOfProjects.scala]: I keep that line before I start working on every PR, but I agree it shouldn't be part of the change so I've removed it now : )

**Comment by xeno-by** [comment]
LGTM
